packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -1517,6 +1517,7 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
     const acceptanceQueue = dumpQueue('acceptanceQueue');
 
     return harden({
+      crankNumber: kvStore.get('crankNumber'),
```

Comment:
no need to change anything, but be aware of an ongoing issue with duplicate cranknums during BOYD+snapshot, #8264